scripts/bin/update-packages

```diff
@@ -367,7 +487,7 @@ _gh_create_new_issue() {
 	fi
 
 	# Extract origin URL, commit hash and builder directory and put everything together
-	link="$(git config --get remote.origin.url |sed -E 's|\.git$||; s|git@([^:]+):(.+)|https://\1/\2|')/blob/$(git rev-parse HEAD)/$(echo */$1)"
+	link="$(git config --get remote.origin.url | sed -E 's|\.git$||; s|git@([^:]+):(.+)|https://\1/\2|')/blob/$(git rev-parse HEAD)/$(echo */"$1")"
```

Comment:
Mostly cosmetic change which potentially fixes problems with non-escaped characters passed there. Can be tested and merged separately.